source/_includes/asides/getting_started_navigation.html

```diff
@@ -27,6 +27,7 @@ <h1 class="title delta">Getting Started Guide</h1>
           <li>{% active_link /getting-started/customizing-devices/ Customizing devices and services %}</li>
           <li>{% active_link /getting-started/presence-detection/ Presence Detection %}</li>
           <li>{% active_link /getting-started/troubleshooting-configuration/ Troubleshooting %}</li>
+          <li>{% active_link /getting-started/securing/ Security Check Points %}</li>
```

Comment:
From my point of view, this is something for the end. It's not required but should be mentioned.